spec/features/views_games_list_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+
+
+feature 'user views restaurants' do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:game1) { FactoryGirl.create(:game1) }
+
+  scenario 'user sees game list' do
+    sign_up
+    game
+    game1
```

Comment:
These lines are a bit confusing. Rather than forcing the use of `let` here I would create the games explicitly:

```ruby
scenario "view game list" do
  first_game = FactoryGirl.create(:game)
  second_game = FactoryGirl.create(:game)
  
  visit "/games"

  expect(page).to have_content(first_game.name)
  expect(page).to have_content(second_game.name)
end
```

You can also simplify this a bit further by creating a list of games:

```ruby
feature "view game list" do
  games = FactoryGirl.create_list(:game, 3)

  visit "/games"

  games.each do |game|
    expect(page).to have_content(game.title)
  end
end
```